ttyd/umbrel-app.yml

```
@@ -0,0 +1,23 @@
+manifestVersion: 1
+id: ttyd
+category: DEVELOPMENT
```

Needs to be `Development` to show properly in the app store

```suggestion
category: Development
```